sockshop/catalogue/data/dump.sql

```diff
@@ -0,0 +1,79 @@
+CREATE USER IF NOT EXISTS 'catalogue_user' IDENTIFIED BY 'default_password';
+
+GRANT ALL ON socksdb.* TO 'catalogue_user';
+
```

Comment:
Not like you have to address on this PR but I want to say that stuff like this (creating DB users and such) should not be in the migration scripts but instead should be an injectable configuration.